src/constant/frogtoberfestIgnoreLabels.js

```diff
@@ -0,0 +1,2 @@
+// Ignore list of labels for frogtoberfest, add at end of array for more list
+export const FROGTOBERFEST_IGNORE_LABEL_LIST = ['invalid'];
```

Comment:
Can we use this where it's needed? It's better not to extract it away to a different file right away. Also, I see it only used once.